conf/report/log-template.html

```diff
@@ -11,6 +11,7 @@ <h3><a href="{{runner_url}}" target="_blank">{{runner}}</a></h3>
 description: {{description|e}}
 rc: {{rc|e}} (means success: {{tool_success|e}})
 should_fail: {{should_fail|e}}
+fail_reason: {{should_fail_because|e}}
```

Comment:
I think that we could check here is `should_fail_because` is an empty string by using something like
```
{% if variable|length %}
    variable is not empty
{% else %}
    variable is empty
{% endif %}
```
and if it's not empty then show `fail_reason` but if it's empty then display old `should_fail` information